docs/source/optional_plugins/varianter_cit.rst

```diff
@@ -0,0 +1,180 @@
+====================
+CIT Varianter Plugin
+====================
+
+This plugin is an implementation of a Combinatorial Interaction
+Testing algorithm for the Avocado varianter functionality.  It
+generates an optimal number of variants, which in turn become
+different test scenarios.
```

Comment:
Maybe we should mention the output is not reproducible and changes on every execution (unlike pict, IIRC)

Reply:
Yep, added: 

```
.. note:: The exact variants generated are not guaranteed to be the same
          across executions.
```

Reply:
That's a feature and it should be documented as such... `To increase the accumulated test coverage, each execution will generate a different set of variants` or something like that. Saying they `are not guaranteed` gives the false impression that it would be good if they were the same across executions, which is not true regarding the combinatorial testing principles.

Reply:
Yep, I had the same impression. Also it'd be perhaps useful to allow overriding the `seed` so one can reproduce it (if necessary).